src/components/Toast/README.md

```diff
@@ -68,18 +68,26 @@ class EmbeddedAppToastExample extends React.Component {
 
 Toast should:
 
-- Be used for short messages to confirm an action. Maximum of 2 lines of text.
-- Not be used for actionable links or messages.
-- Not be used for error messages.
-- Be displayed once at the time. If you need multiple toasts, queue them.
+- Be used for short messages to confirm an action.
+- Not go over 3 words.
+- Rarely be used for error messages.
+
+When to use:
+
+- For success messages
+- Only for non-critical errors that are relevant in the moment and can be explained in 3 words. For example, if there’s an internet connection issue, the toast would say, [Internet disconnected].
```

Comment:
```suggestion
- Only for non-critical errors that are relevant in the moment and can be explained in 3 words. For example, if there’s an internet connection issue, the toast would say, Internet disconnected.
```

Reply:
Ah ok, thank you. I wasn't sure how to treat that.